fbw-common/src/wasm/systems/systems/src/apu/mod.rs

```diff
@@ -367,6 +368,46 @@ impl SimulationElement for AuxiliaryPowerUnitOverheadPanel {
     }
 }
 
+pub struct AirStarterUnit {
+    bleed_air_pressure: Pressure,
+    bleed_air_temperature: ThermodynamicTemperature,
+    turned_on_id: VariableIdentifier,
+    turned_on: bool,
+}
+impl AirStarterUnit {
+    pub fn new(context: &mut InitContext) -> AirStarterUnit {
+        AirStarterUnit {
+            bleed_air_pressure: Pressure::new::<psi>(14.7),
+            bleed_air_temperature: ThermodynamicTemperature::new::<degree_celsius>(15.),
+            turned_on_id: context.get_identifier("ASU_TURNED_ON".to_owned()),
+            turned_on: false,
+        }
+    }
+
+    pub fn update(&mut self) {
+        if self.turned_on {
+            self.bleed_air_pressure = Pressure::new::<psi>(50.0);
+            self.bleed_air_temperature = ThermodynamicTemperature::new::<degree_celsius>(165.);
+        } else {
+            self.bleed_air_pressure = Pressure::new::<psi>(14.7);
```

Comment:
This should probably be the outside static pressure else you'll provide a bleed air source in air when no engine/APU bleed source is available.